coldfront/core/project/forms_/new_project_forms/request_forms.py

```diff
@@ -283,6 +283,44 @@ def _get_form_class(computing_allowance):
 
 class SavioProjectICAExtraFieldsForm(SavioProjectExtraFieldsForm):
 
+    course_name = forms.CharField(
+        help_text=(
+            'Specify the full name of the course, e.g. '
```

Comment:
Inconsistent use of "e.g.":
- 'e.g. CHEM' here
- 'e.g "Dept.' below
- '(e.g. brc-hpc' below

We consistently use "(e.g., a, b, c)"  in user-facing text and throughout the repo.

Reply:
Missing a period at the end of this sentence and the one for `course_department`.